docs/concepts/protocol/07-airstreams.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: "airstreams"
+sidebar_position: 7
+title: "Airstreams"
+---
+
+:::note
+
+Please refer to the [airdrop section](/concepts/use-cases#airdrops) of our use-cases page for an explanation regarding
+why streaming an airdrop makes sense.
+
+:::
+
+## What are Airstreams?
+
+The gist of Airstreams is that instead of airdropping the entirety of the tokens all at once, airdrop recipients receive
+a fraction of the tokens every second through a token stream.
+
+The beauty of it is that airdrop recipients are forced to think long-term and keep the project’s future as their first
+and foremost priority. They are forced to, because instead of receiving all the tokens at once, they receive them over
+time.The gorgeous Airstreams interface
+
+:::info
+
+An airstream can have a duration of a few days, weeks, months and even years. You could, for example, vest the airdrop
+of your new token over 4 years. Or 6 months. You get to choose.
+
+:::
+
+Airstreams not only create the right incentives, but also ensure that the price of your token won’t crash on day one.
+This has been the case for many airdrops, where a large percentage of the airdrop recipients dumped immediately after
+claiming their tokens.
+
+With Airstreams, airdrop recipients only receive a fraction of the tokens every second, and have to manually claim the
+tokens which have already been streamed.
+
+## How It Works
+
+Airstreams can be linear (recipients receive the same amount every second) or exponential (recipients receive
+exponentially more over time)… You choose.
+
+Thanks to our battle-tested token streaming/distribution protocol, you can quickly create Airstreams for thousands of
+recipients in a few clicks using our interface. Recipients and their respective airdrop allocations can be added by
+uploading a simple CSV spreadsheet to the interface
+
+Our spreadsheet feature is the perfect fit for airstreams: it allows you to upload a CSV file with tens of thousands of
+recipients and the interface will automatically create a new airstream for each of these recipients.
+
+Another great advantage, is that creating an Airstream campaign with thousands of recipients won't ruin you in terms of
+gas fees, on the contrary. When launching a campaign, a contract is deployed, and streams are only created when the
+Airstreams are claimed by the Airstream recipients. This is done using a system called _Merkle Distributor_, which is
+backed by a Merkle Tree.
+
+:::note
+
+The contract that implements an Airstream Campaign is called `MerkleStreamer`. For those interested in the technical
+part, [click here](/contracts/v2/reference/periphery/contract.SablierV2MerkleStreamerLL).
+
+:::
+
+When you create an Airstreams campaign, all you are really doing is creating a contract that allows for the recipients
```

Comment:
should be "create an Airstream campaign"

Reply:
It's just grammatical error: "an Airstreams" vs "an Airstream". As I see its _ create an Airstream"s" campaign_

Reply:
oh, plurar, sorry, missed this